lstchain/calib/camera/calibration_calculator.py

```diff
@@ -76,8 +76,8 @@ class CalibrationCalculator(Component):
     ).tag(config=True)
 
     hg_lg_ratio = traits.Float(
-        17.4,
-        help='HG/LG ratio applied if use_scaled_low_gain is True'
+        1.,
+        help='HG/LG ratio applied if use_scaled_low_gain is True. In case of calibrated data the ratio should be 1.'
```

Comment:
Do we have a way to tell automatically from the input?

Reply:
Sorry, I am not sure to understand the question... actually I am in doubt to take this trailet out, it is very unlikely that for calibrated waveforms we need to perform a scaling here...

Reply:
It says "In case of calibrated data", so my question was if we can determine if calibration has already been applied instead of the user having to set this configuration.